cpp/src/IceStorm/Parser.h

```diff
@@ -10,30 +10,6 @@
 #include <list>
 #include <stdio.h>
 
-//
-// Stuff for flex and bison
-//
-
-#define YYSTYPE std::list<std::string>
-#define YY_DECL int yylex(YYSTYPE* yylvalp)
-YY_DECL;
-
-//
-// I must set the initial stack depth to the maximum stack depth to
-// disable bison stack resizing. The bison stack resizing routines use
-// simple malloc/alloc/memcpy calls, which do not work for the
-// YYSTYPE, since YYSTYPE is a C++ type, with constructor, destructor,
-// assignment operator, etc.
-//
-#define YYMAXDEPTH 10000
-#define YYINITDEPTH YYMAXDEPTH // Initial depth is set to max depth, for the reasons described above.
-
-//
-// Newer bison versions allow to disable stack resizing by defining
-// yyoverflow.
-//
-#define yyoverflow(a, b, c, d, e, f) yyerror(a)
```

Comment:
These definitions were moved into `Grammar.y`, since that's where we actually use them.
This is also where they live for the Slice compiler, so it's more consistent as well.

`Grammar.y` `include`s this file at the top, so moving it shouldn't of affected anything.